docs/examples/advancedChoiceWorld.m

```diff
@@ -122,9 +122,13 @@ function advancedChoiceWorld(t, evts, p, vs, in, out, audio)
 evts.azimuth = azimuth;
 evts.response = response;
 evts.feedback = feedback;
+evts.reward = reward;
```

Comment:
Why add this to events?  Don't forget that the times and values of the outputs are automatically saved...

Reply:
hmm yeah I was looking at something with this, but forget exactly what. I'll remove it